rpc/amqp-rpc-connection_test.go

```diff
@@ -0,0 +1,130 @@
+package rpc
```

Comment:
nit: this file name is really long. can it just be connection_test.go? similarly with its originator.